services/bundle_analysis/new_notify/contexts/comment.py

```diff
@@ -0,0 +1,156 @@
+import logging
+from typing import Literal
+
+from asgiref.sync import async_to_sync
+from shared.bundle_analysis import (
+    BundleAnalysisComparison,
+)
+from shared.yaml import UserYaml
+
+from database.models.core import Commit
+from services.bundle_analysis.comparison import ComparisonLoader
+from services.bundle_analysis.exceptions import (
+    MissingBaseCommit,
+    MissingBaseReport,
+    MissingHeadCommit,
+    MissingHeadReport,
+)
+from services.bundle_analysis.new_notify.contexts import (
+    BaseBundleAnalysisNotificationContext,
+    NotificationContextBuilder,
+    NotificationContextBuildError,
+)
+from services.bundle_analysis.new_notify.types import NotificationType
+from services.repository import (
+    EnrichedPull,
+    fetch_and_update_pull_request_information_from_commit,
+)
+
+log = logging.getLogger(__name__)
+
+
+class BundleAnalysisCommentNotificationContext(BaseBundleAnalysisNotificationContext):
```

Comment:
Optional: since we call the notification type `PR_COMMENT`, would it make sense to call this `BundleAnalysisPRCommentNotificationContext` comment instead?